### DISCUSSION TITLE
Timers

### DISCUSSION THREAD

**Discussion by avvertix** [tmitime/tmi#10] [Ideas]
Adding tasks by hand could be useful, but what about forgetting when you started working on it. Usually time trackers have a timer feature you could start and stop so it will count the exact minutes spent on that activity. 

This could be a nice addition especially if the timer can be started on one device and stopped on another.

The timer could be bound to a project and auto-stop after X hours, if you forget to. 

Thinking in perspective of external integrations the timer can be started by an action on a third party system, like entering in a meeting, being assigned a review or simply by sending events.

**Comment by AlbinaMuzafarova** [comment]
sounds like one of the priority issues for me, since it's a lot easier to forget when you started an activity than stopping the timer ;) 

I find it more intuitive to have the timer available on activity level, which are by default project bound. But also project level buttons would probably work. 

As for integrations, sounds like a fancy but nice-to-have option.
